draft-ietf-quic-http.md

```diff
@@ -153,11 +156,11 @@ client making the indicated request.  This trades off network usage against a
 potential latency gain.  Several HTTP/3 frames are used to manage server push,
 such as PUSH_PROMISE, MAX_PUSH_ID, and CANCEL_PUSH.
 
-As in HTTP/2, request and response headers are compressed for transmission.
+As in HTTP/2, request and response fields are compressed for transmission.
 Because HPACK {{?HPACK=RFC7541}} relies on in-order transmission of compressed
-header blocks (a guarantee not provided by QUIC), HTTP/3 replaces HPACK with
+field blocks (a guarantee not provided by QUIC), HTTP/3 replaces HPACK with
```

Comment:
"field sections" is the term of art - see https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#header.fields
```suggestion
field sections (a guarantee not provided by QUIC), HTTP/3 replaces HPACK with
```

Reply:
The problem is that "header blocks" was an HPACK term of art:  The sequence of compressed bytes containing the header set.  I think in this section we need to do QPACK first, then apply the appropriate terms to H3 once we've got them straight over there.

Reply:
There is no concept of header (or trailer) blocks in QPACK -- it doesn't split things up into multiple frames like HPACK/H2 did.

Reply:
e92c94f matches the removal of "blocks" from #3520.